Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -161,6 +170,10 @@ namespace SandboxEditor
                     }
                     eventType = MouseEvent::Down;
                 }
+                m_captureStart = true;
+                AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Event(
+                    GetViewportId(), &AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Events::SetCursorMode, AzToolsFramework::CursorInputMode::CursorModeWrapped);
+          
```

Comment:
update: I think all this code needs to be wrapped around the `ManipulatorMouseWrap` check right? We only want to do this if that's enabled?

Reply:
yep, this was just a rough implementation that shows that it works.

Reply:
Got you, just give me a shout when this is ready for review again, thanks!